cost/azure/hybrid_use_benefit_sql/CHANGELOG.md

```diff
@@ -3,3 +3,9 @@
 ## v1.0
 
 - Initial release
+
+## v2.0
+
+- Includes Azure SQL databases, SQL Virtual machines, SQL Managed Instances
+- detects eligible instances
+- AHUB can automatically be applied
```

Comment:
Usually newer versions should be higher up in the ChangeLog so that they are easier to find.